AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -18,6 +18,42 @@ struct SearchView: View {
         VStack(alignment: .leading) {
             title
             textField
+            parametersQuestion
+            HStack(spacing: 12) {
+                ForEach(ParameterType.allCases) { param in
```

Comment:
this is an interesting technique for `enum` usage, but let's stick to the `ViewModel` approach so we don't mix 10 different coding styles across the project. I also think it would be more readable if we just had `viewModel.searchParameters` exposed:
```Swift
string SearchParameter {
  let isSelected: Bool
  let name: String
}
```